src/include/daos/pool_map.h

```diff
@@ -245,7 +245,8 @@ static inline bool
 pool_target_unavail(struct pool_target *tgt)
```

Comment:
add a 'for_reint' parameter, and regard 'UP' as available when 'for_reint' is true.